tests/Unit/CssInlinerTest.php

```diff
@@ -3140,6 +3140,97 @@ public function inlineCssNotInDebugModeIgnoresOnlyInvalidExcludedSelector(): voi
         self::assertStringContainsString('<p class="z"></p>', $result);
     }
 
+    /**
+     * @test
+     *
+     * @dataProvider excludedCssRuleDataProvider
+     *
+     * @param string $css
+     * @param array<string> $exludedSelectors
+     * @param string $expectedInlineCss
+     */
+    public function addExcludedCssSelectorCanExcludeCssSelectors(
+        string $css,
+        array $exludedSelectors,
+        string $expectedInlineCss
+    ): void
+    {
```

Comment:
I think the `braces_position` error is coming from here - it wants `): void {` all on one line.

Reply:
Though TBH, I'd prefer it as you have it - only put the function opening brace on the same line as the closing argument brace if there's no type annotation.  But that's beyond the scope of this.

Reply:
I ran into issues while trying to run it locally: 
```
  The rules contain unknown fixers: "type_declaration_spaces", "spaces_inside_parentheses" (did you mean "no_spaces_inside_parenthesis"?).
````

Reply:
> I ran into issues while trying to run it locally:

Sounds like some of the development tools (e.g. PHP CS Fixer) need updating.  [The `CONTRIBUTING` file is out of date](https://github.com/MyIntervals/emogrifier/issues/373).  Some tools are now installed with PHIVE.  Seems like you have previously installed them with PHIVE (otherwise I wouldn't expect them run at all), and just need to run `phive update` from the project root directory.